src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -17,6 +22,37 @@
 
 /** Base for the DB filters / conditions that we want to use with queries */
 public abstract class DBFilterBase implements Supplier<BuiltCondition> {
+  /** Filter condition element path. */
+  private final String path;
+
+  protected DBFilterBase(String path) {
+    this.path = path;
+  }
+
+  /**
+   * Get JsonNode for the representing filter condition value.
+   *
+   * @param nodeFactory
+   * @return
```

Comment:
Forgot to suggest that Javadoc should explain what returning `null` means here (not to be included in.... default value sets?)